### PR TITLE
Fix Revealer Qt layout

### PR DESCRIPTION
**Pull request by sorenstoutner**
This fixes the layout problem discussed at #8586.

### CODE REVIEW

**Comment by sorenstoutner** [comment]
Thanks for explaining what the underscore before a string indicates.  I wondered what that meant but didn't take the time to explore it.

I do understand the desire not to make trivial changes to translated strings.  Although I think the newer version looks better (the main reason I made the change was that the dialog looked weird with a lowercase "e" at the beginning of the second part of the string; I just altered the punctuation to be more expressive since I was in there anyway), I can see this not raising to the level of instantiating a translation change, so I will revert it to the old string.

As a little bit of background, when I started working on these dialogs my intention was only to fix the spacing and clipping of the text.  But once I got into it I realized that the word wrapping text, particularly those that included <br/>, let to problematic layouts on high DPI monitors.  So, I needed to separate those string into two new strings.  After I had done that, I realized that the existing text was so confusing that it took experimentation with the dialog for me to even understand how to use it.  So I rewrote a bunch of the strings to be more expressive for a first-time user.  By that point I had touched almost all the strings, so cleaning up this last one seemed like a natural way to go.

You will notice that, to make the layout adapt correctly, I have removed all the custom WWLabel  (Word Wrap Label) widgets and replaced them with standard QLabels.

File: electrum/plugins/revealer/qt.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 Do you have something to hide?
 Secret backup plug-in for the electrum wallet.
 
-Tiago Romagnani Silveira, 2017
-
+Copyright:
+    2017 Tiago Romagnani Silveira
+    2023 Soren Stoutner <soren@stoutner.com>
```
Comment (SomberNight) [inline electrum/plugins/revealer/qt.py:9]: Now that you've also added the word "copyright", I would feel better if we explicitly specified the license.
Before your change, I would say it was implicitly MIT, inherited from the root directory license.

You can just add e.g.
```
Distributed under the MIT software license, see the accompanying
file LICENCE or http://www.opensource.org/licenses/mit-license.php
```
Or I can do it too if you like, just comment that it's ok :)

Comment (sorenstoutner) [reply electrum/plugins/revealer/qt.py:9]: That's a good Idea.  I'm a big fan of all files explicitly stating both their copyright and their license information.

**Comment by SomberNight** [comment]
Thanks

**Pull request merged**